tests/test_derivatives.py

```diff
@@ -689,6 +689,18 @@ def test_dxdy(self):
         assert Add(*term0.expand().args) == term1.expand()
 
 
+class TestCallAPI(object):
```

Comment:
I don't mind that but I think we have some test here and there from previous bugs (like `x0=...`) that should belong there as well then.

Blank line between class def and first docstring

Reply:
Which tests did you have in mind?

Reply:
`test_fd_new_order`, `test_fd_new_side`, `test_fd_new_x0`, `test_fd_new_lo` there is quite a few so I would probably just move this with those in the normal `TestFD`. This actually probably belongs in `test_fd_new_order` as a `parametrized` case

Reply:
Moved it up to be with the other tests, and added a similar test for x0 with x-derivs